mysql-test/suite/binlog/t/binlog_gtid_recovery.test

```diff
@@ -0,0 +1,40 @@
+# PS-9132 mysql.gtid_executed persistent GTID info lost when MySQL crash in Gtid_state::save
+#
+--source include/have_log_bin.inc
```

Comment:
Please source `include/have_debug.inc` to make it skip when run with release builds.

Reply:
Done.